editors/neovim/Portfile

```diff
@@ -37,7 +35,8 @@ depends_lib             port:gettext \
                         path:lib/libluajit-5.1.2.dylib:luajit \
                         port:lua51-lpeg \
                         port:lua51-mpack \
-                        port:luv-luajit
+                        port:luv-luajit \
+                        port:tree-sitter
```

Comment:
Can you post the output of `otool -L /opt/local/bin/nvim`? Last time I checked, nvim is linked to libiconv.2.dylib from `port:libiconv`.

Reply:
```
/opt/local/bin/nvim:
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1775.118.101)
	/opt/local/lib/libintl.8.dylib (compatibility version 10.0.0, current version 10.5.0)
	/opt/local/lib/libiconv.2.dylib (compatibility version 9.0.0, current version 9.1.0)
	/opt/local/lib/libluv.1.dylib (compatibility version 1.0.0, current version 1.41.0)
	/opt/local/lib/libuv.1.dylib (compatibility version 2.0.0, current version 2.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
	/opt/local/lib/libmsgpackc.2.dylib (compatibility version 2.0.0, current version 2.0.0)
	/opt/local/lib/libvterm.0.dylib (compatibility version 1.0.0, current version 1.4.0)
	/opt/local/lib/libtermkey.1.dylib (compatibility version 16.0.0, current version 16.2.0)
	/opt/local/lib/libunibilium.4.dylib (compatibility version 5.0.0, current version 5.0.0)
	/opt/local/lib/libtree-sitter.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libutil.dylib (compatibility version 1.0.0, current version 1.0.0)
	/opt/local/lib/libluajit-5.1.2.dylib (compatibility version 2.1.0, current version 2.1.0)
```

Reply:
It seems `port:libiconv` is depended by `port:gettext`, which is then depended by `port:neovim`.

https://github.com/macports/macports-ports/blob/689c36329bc442db0a62516aa645297a9e92019b/devel/gettext/Portfile#L50

Reply:
> It seems `port:libiconv` is depended by `port:gettext`, which is then depended by `port:neovim`.

Yes, but for the sake of revision bump convenience, please add all linked libraries to depends_lib.